src/middleware/updateValidation.js

```diff
@@ -0,0 +1,46 @@
+const constants = require('../lib/constants');
+
+// eslint-disable-next-line consistent-return
+const validateUpdate = (req, res, next) => {
```

Comment:
I have a question in this middleware. Is this middleware made to prevent the user from updating a single field in the user profile, instead making them update the whole information?

Reply:
No , I am using the validation info to ensure that the new information you add to edit a field is compliant with our db standards, in registering we are making sure that the phone number is in a certain form, when you update the phone number it certainly have to be in that form also.